usaspending_api/etl/management/commands/load_table_to_delta.py

```diff
@@ -122,7 +122,7 @@
         "delta_table_create_sql": transaction_fabs_sql_string,
         "source_schema": None,
         "custom_schema": "",
-        "column_names": list(TRANSACTION_FABS_COLUMNS),
+        "column_names": TRANSACTION_FABS_COLUMNS,
```

Comment:
Be careful about how this could affect usages of this field in the `TABLE_SPEC`. `TRANSACTION_FABS_COLUMNS` may now represent a list, but it's a list of `TransactionColumn`s not a list of dictionaries.

Reply:
Actually, in transaction_f[ab|pd]s.py, the list of `TransactionColumn`s are `TRANSACTION_F[AB|PD]S_COLUMN_INFO`, and `TRANSACTION_F[AB|PD]S_COLUMNS` are defined below as lists of dictionaries.

Reply:
Ah cool, I see that now 👍